doc/classes/Texture.xml

```diff
@@ -117,22 +117,25 @@
 	</methods>
 	<constants>
 		<constant name="FLAG_MIPMAPS" value="1" enum="Flags">
-			Generate mipmaps, to enable smooth zooming out of the texture.
+			Generates mipmaps, generates smaller versions of the same texture to use when zoomed out, keeping the ratio. 
 		</constant>
 		<constant name="FLAG_REPEAT" value="2" enum="Flags">
-			Repeat (instead of clamp to edge).
+			Repeats texture (instead of clamp to edge).
```

Comment:
What's the point of the parenthetical? It repeats instead of doing any number of other things like stretching too.